src/molecules/Modal/ModalContent.native.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Space from '../../atoms/Space';
+import View from '../../atoms/View';
+
+const ModalContent = ({ children, type }) => {
+  return type === 'centered' ? (
+    <Space padding={[2]}>
+      <View>{children}</View>
+    </Space>
+  ) : type === 'fullscreen' ? (
+    children
+  ) : type === 'bottomsheet' ? (
```

Comment:
confirm with Nishant once if this the intended behavior. As per my understanding the centerd and bottomsheet variant content should have fixed surrounding padding